src/docker-osx-dev

```diff
@@ -102,6 +102,28 @@ function assert_non_empty {
 }
 
 #
+# Usage: assert_mutually_exclusive VAR_NAME1 VAR_NAME2 ... VAR_NAMEn
+#
+# Asserts that at most one of $VAR_NAME1, $VAR_NAME2, ..., $VAR_NAMEn is defined
+#
+function assert_mutually_exclusive {
```

Comment:
This is nice, but I'm not sure how many other use cases we'll have like it to merit supporting this much code. It seems like a simple `if $only_dependencies && $skip_dependencies; then log_error ...` would be enough.